File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        h = {}
        output = []
        for n in nums:
            h[n] = h.get(n, 0) + 1
        i = 0
        while i < k:
            m = max(h, key = h.get)
            output.append(m)
            h[m] = 0
            i = i + 1
        return output 

File: Data Structures & Algorithms/contains-duplicate-ii/submission-1.java
class Solution {
    public boolean containsNearbyDuplicate(int[] nums, int k) {
        Set<Integer> set = new HashSet<>();
        int left = 0;

        for (int right = 0; right < nums.length; right++) {
            if (right - left > k) {
                set.remove(nums[left]);
                left++;
            }
            if (set.contains(nums[right])) {
                return true;
            }
            set.add(nums[right]);
        }
        return false;
    
    }
}